gradle/dependency-locks/embulkPluginRuntime.lockfile

```diff
@@ -1,30 +0,0 @@
-# This is a Gradle generated file for dependency locking.
```

Comment:
could you re-add this file for tracking dependency with core?

Reply:
Done